app/templates/components/search-result.hbs

```diff
@@ -0,0 +1,21 @@
+<div>
+    <h4><a>{{result.title}}</a></h4>
+    {{#each result.contributors as |contributor|}}
+        <a>{{contributor.familyName}}, {{contributor.givenName}}</a>
+    {{/each}}
+    <p {{action "toggleShowDescription"}} style="cursor: pointer; cursor: hand;">{{displayDescription}}</p>
+
+    <div class="row">
+        <div class="col-sm-6 text-muted">
+            <p>Released on: {{moment-format result.date_updated "YYYY-MM-DD hh:mm a"}}</p>
```

Comment:
Rather than hard-coding date formats in every use of moment.js, we may want to consider defining a custom date format once, to help maintainability/ consistency.
https://github.com/stefanpenner/ember-moment#global-default-output-format